src/Aspire.ProjectTemplates/templates/aspire-empty/9.4/AspireApplication.1.ServiceDefaults/AspireApplication.1.ServiceDefaults.csproj

```diff
@@ -10,18 +10,18 @@
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
 
-    <PackageReference Include="Microsoft.Extensions.Http.Resilience" Version="!!REPLACE_WITH_DOTNET_EXTENSIONS_VERSION!!" />
-    <PackageReference Include="Microsoft.Extensions.ServiceDiscovery" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' == 'net8.0' " Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="!!REPLACE_WITH_OTEL_LTS_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' == 'net8.0' " Include="OpenTelemetry.Extensions.Hosting" Version="!!REPLACE_WITH_OTEL_LTS_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' == 'net8.0' " Include="OpenTelemetry.Instrumentation.AspNetCore" Version="!!REPLACE_WITH_OTEL_LTS_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' == 'net8.0' " Include="OpenTelemetry.Instrumentation.Http" Version="!!REPLACE_WITH_OTEL_LTS_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' == 'net8.0' " Include="OpenTelemetry.Instrumentation.Runtime" Version="!!REPLACE_WITH_OTEL_LTS_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' != 'net8.0' " Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="!!REPLACE_WITH_OTEL_EXPORTER_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' != 'net8.0' " Include="OpenTelemetry.Extensions.Hosting" Version="!!REPLACE_WITH_OTEL_HOSTING_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' != 'net8.0' " Include="OpenTelemetry.Instrumentation.AspNetCore" Version="!!REPLACE_WITH_OTEL_ASPNETCORE_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' != 'net8.0' " Include="OpenTelemetry.Instrumentation.Http" Version="!!REPLACE_WITH_OTEL_HTTP_VERSION!!" />
-    <PackageReference Condition=" '$(Framework)' != 'net8.0' " Include="OpenTelemetry.Instrumentation.Runtime" Version="!!REPLACE_WITH_OTEL_RUNTIME_VERSION!!" />
+    <PackageReference Include="Microsoft.Extensions.Http.Resilience" Version="9.6.0" />
+    <PackageReference Include="Microsoft.Extensions.ServiceDiscovery" Version="9.4.1" />
+    <PackageReference Condition=" '$(Framework)' == 'net8.0' " Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.9.0" />
+    <PackageReference Condition=" '$(Framework)' == 'net8.0' " Include="OpenTelemetry.Extensions.Hosting" Version="1.9.0" />
```

Comment:
aside -- I can't actually find where otel .NET declares what version is LTS

Reply:
It doesn't, we pin to 1.9.0 here due to #7123